docs/news/posts/2024-07-02-release-2.6.0.md

```diff
@@ -0,0 +1,18 @@
+---
+blogpost: true
+category: Releases
+tags: aiida-core
+date: 2024-07-02
+---
+
+# AiiDA v2.6.0 released
+
+We just released the latest version of AiiDA. This minor release comes with a number of features that are focused on user friendliness of the CLI and the API. The [changelog](https://github.com/aiidateam/aiida-core/blob/v2.6.0/CHANGELOG.md) contains a list of all changes including a nice overview of some of the new features.
+
+A lot of work has especially gone in making it easier to install AiiDA and try it out without having to install services like PostgreSQL and RabbitMQ. It is now possible to operate AiiDA without RabbitMQ, at the cost of losing daemon functionality. The new command verdi presto allows setting up a complete and ready-to-go profile with a single command without any configuration. The installation [documentation](https://aiida.readthedocs.io/projects/aiida-core/en/stable/installation/index.html) has seen a complete overhaul.
```

Comment:
`verdi presto` to distinguish the command from surrounding text

Reply:
that is, add tick marks :)

Reply:
```suggestion
A lot of work has especially gone in making it easier to install AiiDA and try it out without having to install services like PostgreSQL and RabbitMQ. It is now possible to operate AiiDA without RabbitMQ, at the cost of losing daemon functionality. The new command `verdi presto` allows setting up a complete and ready-to-go profile with a single command without any configuration. The installation [documentation](https://aiida.readthedocs.io/projects/aiida-core/en/stable/installation/index.html) has seen a complete overhaul.
```